### PR TITLE
config: fix subjects flag, add example for editing global config

### PR DESCRIPTION
**Pull request by bobheadxi**
Currently, providing a `-subject` flag to `src config` does not apply it.

```
+ src config edit -subject=U2l0ZToic2l0ZSI= -property=notices '-value=[]'
GraphQL errors: 1 error occurred:
        * {
  "locations": [
    {
      "column": 39,
      "line": 2
    }
  ],
  "message": "Variable \"subject\" has invalid value null.\nExpected type \"ID!\", found null."
}
```

Also, the docs state that this command can edit global config, but does not imply how, so I added an example with a workaround - let me know if there's a proper way to do this!

### CODE REVIEW

File: cmd/src/config_edit.go [hunk 1]
```diff
@@ -34,6 +34,9 @@ Examples:
 
     	$ src config edit -subject=$(src orgs get -f '{{.ID}}' -name=abc-org) -overwrite -value '{"motd":["Hello!"]}'
 
+  Change global settings:
+		
+        $ src config edit -subject=$(echo 'query { site { id } }' | src api | jq .data.site.id --raw-output)
```
Comment (LawnGnome) [inline cmd/src/config_edit.go:39]: The indentation on this was slightly off compared to the earlier examples:

```suggestion
      $ src config edit -subject=$(echo 'query { site { id } }' | src api | jq .data.site.id --raw-output)
```

**Review by LawnGnome — APPROVED** [review]
Good catch!

File: cmd/src/config_edit.go [hunk 2]
```diff
@@ -34,6 +34,9 @@ Examples:
 
     	$ src config edit -subject=$(src orgs get -f '{{.ID}}' -name=abc-org) -overwrite -value '{"motd":["Hello!"]}'
 
+  Change global settings:
+		
+      $ src config edit -subject=$(echo 'query { site { id } }' | src api | jq .data.site.id --raw-output)
```
Comment (bobheadxi) [inline cmd/src/config_edit.go:39]:
```suggestion
    	$ src config edit -subject=$(echo 'query { site { id } }' | src api | jq .data.site.id --raw-output)
```

File: cmd/src/config_edit.go [hunk 3]
```diff
@@ -34,6 +34,9 @@ Examples:
 
     	$ src config edit -subject=$(src orgs get -f '{{.ID}}' -name=abc-org) -overwrite -value '{"motd":["Hello!"]}'
 
+  Change global settings:
+
```
Comment (bobheadxi) [inline cmd/src/config_edit.go:38]:
```suggestion

```

File: cmd/src/config_edit.go [hunk 4]
```diff
@@ -34,6 +34,8 @@ Examples:
 
     	$ src config edit -subject=$(src orgs get -f '{{.ID}}' -name=abc-org) -overwrite -value '{"motd":["Hello!"]}'
 
+  Change global settings:
```
Comment (bobheadxi) [inline cmd/src/config_edit.go:37]:
```suggestion

  Change global settings:
```

**Pull request merged**